tools/js2c.py

```diff
@@ -261,7 +261,8 @@ def NormalizeFileName(filename):
     split = ['internal'] + split
   else:  # `lib/**/*.js` so drop the 'lib' part
     split = split[1:]
-  filename = '/'.join(split)
+  if split != []:
```

Comment:
No as critical, but should be
```suggestion
  if len(split):
```
as it's more "pythonic", and doesn't assume `split` is a list, only that it's enumerable.